Tutorial12.4/example-1/menu.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>The Net Ninja - CSS Flexbox</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+  </head>
+  <body>
+    <div class="wrapper">
+
```

Comment:
@Anisha-Alexander Install prettier to format the code?

Reply:
Will install .